draft-ietf-httpbis-cache-latest.xml

```diff
@@ -1027,26 +1022,14 @@
    containing an <x:ref>If-Modified-Since</x:ref> header field
    (<xref target="field.if-modified-since"/>) indicates that the client wants to validate
    one or more of its own stored responses by modification date.
-   A cache recipient &SHOULD; generate a <x:ref>304 (Not Modified)</x:ref>
-   response (using the metadata of the selected stored response) if one of the
-   following cases is true:
 </t>
-<ol>
-  <li>
-    the selected stored response has a <x:ref>Last-Modified</x:ref>
-    field value that is earlier than or equal to the conditional timestamp;
-  </li>
-  <li>
-    no <x:ref>Last-Modified</x:ref> header field is present in the selected stored
-    response, but it has a <x:ref>Date</x:ref> field value that is earlier than
-    or equal to the conditional timestamp; or,
-  </li>
-  <li>
-    neither <x:ref>Last-Modified</x:ref> nor <x:ref>Date</x:ref> is present
-    in the selected stored response, but the cache recorded it as having been
-    received at a time earlier than or equal to the conditional timestamp.
-  </li>
-</ol>
+<t>
+   If a request contains the <x:ref>If-Modified-Since</x:ref> header field and
+   the <x:ref>Last-Modified</x:ref> header field is not present in a selected
+   stored response, a cache &SHOULD; use the <x:ref>Date</x:ref> field value
+   to evaluate the conditional; if it is not present, the time that the
+   response was received &SHOULD; be used.
```

Comment:
```suggestion
   If a request contains an <x:ref>If-Modified-Since</x:ref> header field and
   the <x:ref>Last-Modified</x:ref> header field is not present in a selected
   stored response, a cache &SHOULD; use the stored response's
   <x:ref>Date</x:ref> field value (or, if no Date field is present, the time
   that the stored response was received) to evaluate the conditional.
```